tutorials/01_introductory/lifecycle.py

```diff
@@ -0,0 +1,288 @@
+# We can then apply this formatter to the labels on our plot. To do this,
+# we'll use the ``xaxis`` attribute of our axis. This lets you perform
+# actions on a specific axis on our plot.
+
+fig, ax = plt.subplots(figsize=(6, 8))
+ax.barh(group_names, group_data)
+labels = ax.get_xticklabels()
+plt.setp(labels, rotation=45, horizontalalignment='right')
+
+ax.set(xlim=[-10000, 140000], xlabel='Total Revenue', ylabel='Company',
+       title='Company Revenue')
+ax.xaxis.set_major_formatter(formatter)
+
+###############################################################################
+# Combining multiple visualizations
+# =================================
+#
+# It is possible to draw multiple plot elements on the same instance of
+# :class:`axes.Axes`. To do this we simply need to call another one of
+# the plot methods on that axes object.
+
+fig, ax = plt.subplots(figsize=(8, 8))
+ax.barh(group_names, group_data)
+labels = ax.get_xticklabels()
+plt.setp(labels, rotation=45, horizontalalignment='right')
+
+# Add a vertical line, here we set the style in the function call
+ax.axvline(group_mean, ls='--', color='r')
+
+# Annotate new companies
+for group in [3, 5, 8]:
+    ax.text(145000, group, "New Company", fontsize=10,
+            verticalalignment="center")
+
+# Now we'll move our title up since it's getting a little cramped
+ax.title.set(y=1.05)
+
+ax.set(xlim=[-10000, 140000], xlabel='Total Revenue', ylabel='Company',
+       title='Company Revenue')
+ax.xaxis.set_major_formatter(formatter)
+ax.set_xticks([25e3, 50e3, 75e3, 100e3, 125e3])
+fig.subplots_adjust(right=.1)
+plt.show()
+
+
+###############################################################################
+# Saving our plot
+# ===============
+#
+# Now that we're happy with the outcome of our plot, we want to save it to
+# disk. There are many file formats we can save to in Matplotlib. To see
+# a list of available options, use:
+
+print(fig.canvas.get_supported_filetypes())
+
+###############################################################################
+# We can then use the :meth:`figure.Figure.savefig` in order to save the figure
+# to disk. Note that there are several useful flags we'll show below:
+#
+# * ``transparent=True`` makes the background of the saved figure transparent.
+# * ``dpi=80`` controls the resolution (dots per square inch) of the ouput.
+# * ``bbox_inches="tight"`` fits the bounds of the figure to our plot.
+
+# Uncomment this line to save the figure.
+# fig.savefig('sales.png', transparent=False, dpi=80, bbox_inches="tight")
+
+###############################################################################
+# Below we've included a small diagram showing the basic components of the
+# plot elements that were included in the original blog post. It has most of
+# the same elements, with a few extra components as well. Click the image to
+# see the blog post (along with code used to generate the figure).
+#
+# .. image:: ../../_static/axis_layout.png
```

Comment:
I agree that you should drop the figure as it doesn't flow with this tutorial and an accurate one can get folded back in.